.github/dependabot.yml

```diff
@@ -5,6 +5,13 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "gradle"
+    directory: "/"
+    allow:
+      - dependency-name: "com.gradle*"
+    schedule:
+      interval: "monthly"
```

Comment:
i see you are trying to match the monthly cycle of the other checks, but I think for our purposes, weekly might be better.

Reply:
I would adapt to the common frequency for all other updates indeed